tradeit-android-sdk/src/main/java/trade/it/android/sdk/model/TradeItLinkedBrokerCache.java

```diff
@@ -0,0 +1,49 @@
+package trade.it.android.sdk.model;
+
+
+import android.content.Context;
+import android.content.SharedPreferences;
+
+import com.google.gson.Gson;
+
+import java.util.HashSet;
+import java.util.Set;
+
+
+public class TradeItLinkedBrokerCache {
+    private static final String LINKED_BROKER_CACHE_KEY = "TRADE_IT_LINKED_BROKER_CACHE";
+    private static final String TRADE_IT_SHARED_PREFS_KEY = "TRADE_IT_SHARED_PREFS_KEY";
```

Comment:
The shared prefs file name should relate to the sdk package name, so it's clearer where it's from.